tests/Microsoft.Identity.Test.Integration.net45/HeadlessTests/ConfidentialClientIntegrationTests.cs

```diff
@@ -383,27 +412,37 @@ public async Task AcquireTokenWithClientSecretFromAdfsAsync()
             Assert.IsNull(authResult.IdToken);
         }
 
-        private async Task RunOnBehalfOfTestAsync(LabResponse labResponse)
+        private async Task RunOnBehalfOfTestAsync(LabResponse labResponse, bool usGov = false)
```

Comment:
Next ask might be to add tests for Germany cloud. `bool` is not a good choice here, an enum is.

Reply:
I have certainly asked for this previously so not an unreal scenario